### ISSUE TITLE
GUI visual artifacts render while opening Magnet Links 4.0 build fd9c454c45 #8464

### ISSUE THREAD

**Issue by chaseholden** [transmission/transmission#4165]
### What is the issue?

Unexpected GUI visual anomalies render while opening Magnet links, cause temporary visual artifacts for 4.0 dev builds up to build fd9c454c45 #8464

- Attached video capture only includes with collapsed group headers. It does not show the multiple apx 20 magnet URL popup prompts off screen.

- The anomalies or artifacts occur one time for each opening of a magnet link in succession.

- These have never occurred in any version of Transmission 3.0 for me, but have persisted throughout 4.0 dev builds for many months. I never reported before because I presumed this was known and experienced by all.

https://user-images.githubusercontent.com/19694741/201550367-c952324c-b2b1-46ed-a627-e1e56adc8ea3.mp4

### Which application of Transmission?

macOS app for macOS 12.6 Monterey thru 13.1 Ventura for M1 Ultra ARM64

### Which version of Transmission?

4.0 dev build fd9c454c45 #8464

**Comment by ckerr** [comment]
Is this still happening? Is anyone able to reproduce this issue?

**Comment by ckerr** [comment]
Is this still happening? Is anyone able to reproduce this issue?

**Comment by chaseholden** [comment]
@ckerr  In 4.0.2, I just attempted to recreate this and it appears to be solved and fixed. No anomalies even when scripting the opening sequence of 100 magnet links. 

However, Transmission 4.0.2 did crash after the first 5 magnet links or so, then relaunched and I was able to open the remaining 95 or so.

Here's a screenshot of the crash, FWIW. I'll close this issue as solved, however.

<img width="871" alt="Screenshot 2023-03-17 at 9 07 49 PM" src="https://user-images.githubusercontent.com/19694741/226084161-1927613c-e9ed-4a36-9d6a-eb98e4fb99c4.png">
